cypress/support/hostsfunctions.ts

```diff
@@ -21,18 +21,12 @@ export function createAndCheckHost(host_type: string, inventory: string) {
 
   // create host
   cy.clickButton(/^Create host$/);
-  cy.verifyPageTitle('Create Host');
+  cy.verifyPageTitle('Create host');
   cy.getByDataCy('name').type(hostName);
   cy.getByDataCy('description').type('This is the description');
 
   if (host_type === 'stand_alone_host') {
-    cy.getByDataCy('inventory').click();
-    cy.contains('button', 'Browse').click();
-    cy.getModal().within(() => {
-      cy.filterTableBySingleSelect('name', inventory);
-      cy.get(`[data-cy="checkbox-column-cell"] input`).click();
-      cy.contains('button', 'Confirm').click();
-    });
+    cy.singleSelectBy('[data-cy="inventory-form-group"]', inventory);
```

Comment:
there is also cy.singleSelectByDataCy() which we can use here instead of cy.singleSelectBy()

Reply:
So just change line 29 to cy.singleSelectByDataCy('[data-cy="inventory-form-group"]', inventory); ?